src/normalize.js

```diff
@@ -252,8 +252,6 @@ export const createNodes = (entity, ctx, uid) => {
           attributeName,
         });
 
-        entryNode.children = entryNode.children.concat([textNode.id]);
```

Comment:
I don't get this?

Reply:
This is the fix we found for #321 after talking with gatsby developers. Normally we should not mutate the children array directly, and in this case it's causing the issue with content refresh

Reply:
shouldn't we at least push the textNode in it then ?

Reply:
You mean pushing it in the children array?

Reply:
yep

Reply:
Do you think using push instead of concat would make a difference?

From what I understand I think we're not supposed to manage the children array at all. Gatsby's recommended solution is to use the [createParentChildLink](https://www.gatsbyjs.com/docs/reference/config-files/actions/#createParentChildLink) API, but in our case that involves a much bigger rewrite.